synapse/config/server.py

```diff
@@ -289,6 +300,11 @@ def default_config(self, server_name, data_dir_path, **kwargs):
         #  - nyc.example.com
         #  - syd.example.com
 
+        # Send outbound federation requests through a seperate traffic gateway.
```

Comment:
you've misspelled "separate", but in any case, I think this could do with rewording (and the option renamed).

AFAICT, it expects a regular HTTP proxy; could you call it that rather than a "gateway" or a "relay"?